.github/codeql/codeql-config.yml

```
@@ -27,9 +27,6 @@ paths:
   - "auth.config.ts"
   - "next.config.ts"
 
-# Use security and quality queries
+# Use standard security queries
 queries:
-  - uses: security-and-quality
-
-# Disable problematic queries that might conflict
-disable-default-queries: true 
\ No newline at end of file
+  - uses: security-and-quality 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)